config/README.md

```diff
@@ -50,6 +51,17 @@ func New(path string) (Config, error)
 ```
 New takes in a path to a configuration yaml and returns a Configuration.
 
+#### type Health
+
+```go
+type Health struct {
```

Comment:
I'm thinking `s/Health/HealthCheck/g`, a small cost to be a little more explicit.